docs/api/public_php_api_managing_content.md

```diff
@@ -65,6 +65,10 @@ In order to set a Location for your object, you must instantiate a [`LocationCre
 $locationCreateStruct = $locationService->newLocationCreateStruct( 2 );
 ```
 
+#### Location object with access to Content
+    
+You can directly get it by using `$location->getContent()`, it can also be very useful in a Twig via `location.content`. This functionality additionally introduces possibility to specify prioritised languages when loading a Location. Content will be loaded on-demand across result set you are loading (e.g. search and other places you can load several Locations).
```

Comment:
So, I checked the doc and actually wanted to ask you if this part is better placed in `Browsing, finding, viewing` as this page is abut creating/updating data atm.